bundle/manifests/siteconfig.clusterserviceversion.yaml

```diff
@@ -801,6 +801,723 @@ metadata:
+                      "config": {
+                        "server": [
+                          "10.11.12.2",
+                          "1111:2222:3333:4444::1:88"
+                        ]
+                      }
+                    },
+                    "interfaces": [
+                      {
+                        "ipv4": {
+                          "address": [
+                            {
+                              "ip": "10.11.12.20",
+                              "prefix-length": 24
+                            }
+                          ],
+                          "enabled": true
+                        },
+                        "ipv6": {
+                          "address": [
+                            {
+                              "ip": "1111:2222:3333:4444::1:20",
+                              "prefix-length": 64
+                            }
+                          ],
+                          "enabled": true
+                        },
+                        "name": "eno1",
+                        "state": "up",
+                        "type": "ethernet"
+                      },
+                      {
+                        "ipv4": {
+                          "dhcp": false,
+                          "enabled": false
+                        },
+                        "ipv6": {
+                          "dhcp": false,
+                          "enabled": false
+                        },
+                        "name": "eno2",
+                        "state": "up",
+                        "type": "ethernet"
+                      }
+                    ],
+                    "routes": {
+                      "config": [
+                        {
+                          "destination": "0.0.0.0/0",
+                          "next-hop-address": "10.11.12.254",
+                          "next-hop-interface": "eno1",
+                          "table-id": 254
+                        },
+                        {
+                          "destination": "::/0",
+                          "next-hop-address": "1111:2222:3333:4444::1:fe",
+                          "next-hop-interface": "eno1"
+                        }
+                      ]
+                    }
+                  },
+                  "interfaces": [
+                    {
+                      "macAddress": "00:11:22:33:44:51",
+                      "name": "eno1"
+                    },
+                    {
+                      "macAddress": "00:11:22:33:44:52",
+                      "name": "eno2"
+                    }
+                  ]
+                },
+                "role": "master",
+                "rootDeviceHints": {
+                  "deviceName": "/dev/disk/by-path/pci-0000:18:00.0-scsi-0:2:0:0"
+                },
+                "templateRefs": [
+                  {
+                    "name": "ai-node-templates-v1",
+                    "namespace": "siteconfig-operator"
```

Comment:
By default I think this configmap is in open-cluster-management.

Reply:
I think the latest version creates these in the siteconfig-operator namespace.
I tested the latest version on my env it was created in this namespace.
@sakhoury can you confirm?